src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -40,6 +41,9 @@
     @Autowired
     private AmqpBridge bridge;
```

Comment:
maybe we should change this from `bridge` to `amqpBridge`. In any case having the application exposing both should be configurable. We can do that even later.